### ISSUE TITLE
Adding HTML content like Adsense ads

### ISSUE THREAD

**Issue by mariaregan** [paperbits/paperbits-demo#16]
Sir, I would like to add HTML widget like Adsense ads. But, there is no such widget. Also, I want to add code inside <head> tag. I think there is no option for that. Please help me.

**Comment by azaslonov** [comment]
Hi @mariaregan. thank you for feedback. I think that AdSense widget would be a good addition to to the list, we'll include it into our scope of works.

Regarding the code inside a tag: one of the key concepts in Paperbits is a separation of data (JSON) from its presentation (HTML, JS, CSS). You can think of it as Markdown, but more sophisticated (since we render not only text but also layouts, widgets, styles, etc.). However, nothing prevents you from creating a widget accepting free HTML/scripts as configuration.

Please check back to the documentation and paperbits-demo repo later this week, we're going to add "How to create a widget" guide and a widget scaffold.

**Comment by azaslonov** [comment]
@mariaregan, as I mentioned, the article is coming, but the scaffold [has been added](https://github.com/paperbits/paperbits-demo/commit/de5229031b146825e5ec111b617e32229f0add93) already. If you pull the latest changes and launch it, you will see it in "Add widget" list.

**Comment by mariaregan** [comment]
Thank you sir. But, I tried and it displays "this is your widget editor
that is yet to be implemented."

**Comment by azaslonov** [comment]
That's right, it's just a scaffold for your own widget where you can replace [HTML content](https://github.com/paperbits/paperbits-demo/blob/master/src/components/your-widget/ko/yourWidget.html) with your own, e.g. with the AdSense script. Also, please read about [Widget Anatomy](https://paperbits.io/wiki/widget-anatomy) to better understand how it works.
